Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php

```diff
@@ -227,4 +224,42 @@ private function debugBuildAllControls(ControlBuilder $builder): ControlBuilder
 		return $builder;
 	}
 
+	private function getTitleByType(string $type): string
+	{
+		switch ($type) {
+			case "file":
+				$title = $this->lng->txt("obj_file");
+				break;
+			case "copa":
+				$title = $this->lng->txt("obj_copa");
+				break;
+			case "lm":
+				$title = $this->lng->txt("obj_lm");
+				break;
+			case "html":
+				$title = $this->lng->txt("obj_htlm");
+				break;
+			case "sahs":
+				$title = $this->lng->txt("obj_sahs");
+				break;
+			case "iass":
+				$title = $this->lng->txt("obj_iass");
+				break;
+			case "exc":
+				$title = $this->lng->txt("obj_exc");
+				break;
+			case "tst":
+				$title = $this->lng->txt("obj_tst");
+				break;
+			case "svy":
+				$title = $this->lng->txt("obj_svy");
+				break;
+			default:
+				$title = "";
+				break;
+		}
+
+		return $title;
```

Comment:
Why not just use `return $this->lng->txt("obj_$type");` here? This IMO would work for plugins too.